handbook/engineering/engineering.rituals.yml

```
@@ -1,19 +1,12 @@
 -
-  task: "Pull request review" # Title that will actually show in rituals table
-  startedOn: "2023-08-09" # Needs to align with frequency e.g. if frequency is every thrid Thursday startedOn === any third thursday
-  frequency: "Daily" # must be supported by
-  description: "Engineers go through pull requests for which their review has been requested." # example of a longer thing: description: "[Prioritizing next sprint](https://fleetdm.com/handbook/company/communication)"
-  moreInfoUrl: "https://fleetdm.com/handbook/company/why-this-way#why-make-work-visible" #URL used to highlight "description:" test in table
-  dri: "lukeheath" # DRI for ritual (assignee if autoIssue) (TODO display GitHub proflie pic instead of name or title)
-  #autoIssue:   # Enables automation of GitHub issues
-    #labels: [ "#g-cx" ] # label to be applied to issue
--
-  task: "Engineering group discussions"
-  startedOn: "2023-08-09" 
-  frequency: "Daily"
-  description: "Engineers go through pull requests for which their review has been requested."
-  moreInfoUrl: 
+  task: "Renew Apple certificate signing request (CSR)"
+  startedOn: "2024-09-01" 
+  frequency: "Annual"
```

```suggestion
  frequency: "Annually"
```

Build is erroring looking for this.